db/migrate/20151119100750_change_description_column_types.rb

```diff
@@ -0,0 +1,19 @@
+class ChangeDescriptionColumnTypes < ActiveRecord::Migration
+  def change
+    [
+      [:draft_content_items, DraftContentItem],
+      [:live_content_items, LiveContentItem],
+    ].each do |table, model|
+      add_column table, :new_description, :json, default: { value: nil }
```

Comment:
Given the discussions we've had about migrations which lock tables, and the advice here: https://www.braintreepayments.com/blog/safe-operations-for-high-volume-postgresql/ we should probably split this into two actions.